sos/report/plugins/system.py

```diff
@@ -23,6 +23,14 @@ def setup(self):
             "/etc/sysconfig",
             "/etc/default",
             "/etc/environment",
+            "info -o vd",
+            "info -o pd",
+            "info -o hba",
+            "arcconf getconfig 1",
+            "smart -p 0",
+            "sas3ircu list",
+            "storcli show all",
+            "storcli /cx[/eall]/sall show",
```

Comment:
If I'm not mistaken, `storcli` is the replacement for `megacli`, in which case these should either go into the `megacli` plugin, or a new `storcli` plugin should be created to house them. In either case, they don't belong in `system`.